personalities/ct-static-api/configpb/config.proto

```diff
@@ -22,65 +22,34 @@ import "crypto/keyspb/keyspb.proto";
 
-message LogBackend {
-  // name defines the name of the log backend for use in LogConfig messages and must be unique.
-  string name = 1;
-  // backend_spec defines the RPC endpoint that clients should use to send requests
-  // to this log backend. These should be in the same format as rpcBackendFlag in the
-  // CTFE main and must not be an empty string.
-  string backend_spec = 2;
-}
-
-// LogBackendSet supports a configuration where a single set of frontends handle
-// requests for multiple backends. For example this could be used to run different
-// backends in different geographic regions.
-message LogBackendSet {
-  repeated LogBackend backend = 1;
-}
-
 // LogConfigSet is a set of LogConfig messages.
 message LogConfigSet {
   repeated LogConfig config = 1;
 }
 
 // LogConfig describes the configuration options for a log instance.
 //
-// NEXT_ID: 22
+// NEXT_ID: 15
 message LogConfig {
-  // The ID of a Trillian tree that stores the log data. The tree type must be
-  // LOG for regular CT logs. For mirror logs it must be either PREORDERED_LOG
-  // or LOG, and can change at runtime. CTFE in mirror mode uses only read API
-  // which is common for both types.
-  int64 log_id = 1;
-  // prefix is the name of the log. It will come after the global or
-  // override handler prefix. For example if the handler prefix is "/logs"
-  // and prefix is "vogon" the get-sth handler for this log will be
-  // available at "/logs/vogon/ct/v1/get-sth". The prefix cannot be empty
-  // and must not include "/" path separator characters.
-  string prefix = 2;
-  // override_handler_prefix if set to a non empty value overrides the global
-  // handler prefix for an individual log. For example this field is set to
-  // "/otherlogs" then a log with prefix "vogon" will make it's get-sth handler
-  // available at "/otherlogs/vogon/ct/v1/get-sth" regardless of what the
-  // global prefix is. Can be set to '/' to make the get-sth handler register
-  // at "/vogon/ct/v1/get-sth".
-  string override_handler_prefix = 13;
+  // origin identifies the log. It will be used in its checkpoint, and
+  // is also its submission prefix, as per https://c2sp.org/static-ct-api
+  string origin = 1;
   // Paths to the files containing root certificates that are acceptable to the
-  // log. The certs are served through get-roots endpoint. Optional in mirrors.
-  repeated string roots_pem_file = 3;
-  // The private key used for signing STHs etc. Not required for mirrors.
-  google.protobuf.Any private_key = 4;
-  // The public key matching the above private key (if both are present). It is
-  // used only by mirror logs for verifying the source log's signatures, but can
-  // be specified for regular logs as well for the convenience of test tools.
-  keyspb.PublicKey public_key = 5;
+  // log. The certs are served through get-roots endpoint.
+  repeated string roots_pem_file = 2;
+  // The private key used for signing STHs etc.
+  google.protobuf.Any private_key = 3;
```

Comment:
I imagine you're going to need some sort of `note.Signer` compatible wrapper around this, presumably you can't go "full-note" since you need to sign SCTs in 6962 stylee.